sanity/schemas/post.js

```diff
@@ -41,6 +41,7 @@ export default {
       name: 'publishedAt',
       title: 'Published at',
       type: 'datetime',
+      validation: Rule => Rule.required(),
```

Comment:
Actually, I also found the `initialValue` templates docs: https://www.sanity.io/docs/initial-value-templates#dec5ffddbcc2 so maybe this is better and then we don't need to validate?

`initialValue: (new Date()).toISOString()`

LMK what you feel is best dev UX here (happy to update PR). I actually think defaulting to "now" might be better but I'll wait to hear if you agree :)